spatialos-sdk/src/worker/op.rs

```diff
@@ -482,8 +479,35 @@ pub struct ReserveEntityIdsResponseOp {
 
 #[derive(Debug)]
 pub struct ReservedEntityIdRange {
-    pub first_entity_id: EntityId,
-    pub number_of_entity_ids: u32,
+    current: i64,
+    consumed: u32,
+    reserved: u32,
+}
+
+impl ReservedEntityIdRange {
```

Comment:
Why not instead use `std::ops::Range` and delete the custom type entirely?

Reply:
Because I didn't know that existed 😂

Reply:
It seems very appropriate - will refactor

Reply:
Sadly we lose the `Iterator` impl as `EntityId` must implement [`Step`](https://doc.rust-lang.org/std/iter/trait.Step.html) to get the blanket implementation for `Range<EntityId>`.

Unfortunately, `Step` is nightly and I don't want to force us onto a nightly version

Reply:
What I can do - is add a TODO to replace with `Range<EntityId>` once `Step` is stabilised.